platform/security/core/security-core-impl/src/main/java/ddf/security/config/impl/ConfigurationSecurityLogger.java

```diff
@@ -73,8 +81,17 @@ public void configurationEvent(ConfigurationEvent event) {
   }
 
   @SuppressWarnings("squid:S00112" /* Generic RuntimeException is acceptable here */)
-  private String dictionaryToString(Dictionary<String, Object> properties) {
+  @VisibleForTesting
+  String dictionaryToString(Dictionary<String, Object> properties) {
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+    if (properties.remove(PASSWORD) != null) {
```

Comment:
❓ How do we know these are the only 2 password properties that should be sanitized? I though this info was stored in the meta types.

Reply:
It is and there could be other properties in the future that have a different name. Currently, all the passwords for sources and LDAP that are being printed in the security lo have either `password` or `ldapBindUserPass ` as their key.

Reply:
There is already a metatype service that you could use to look up which properties are of the password type and exclude them based on that, rather than coupling this to the properties that we currently have. It is inevitable that this will get out of sync. However, I'm not entirely sure why we're auditing the entire configuration change in the first place. We should audit that the configuration changed, but I'm not sure that I care what the values are. If we do have some compelling reason to keep auditing the actual values, then I think we should hook it up to the other metatype service instead.

Reply:
We are no longer auditing the entire configuration just the type (update, create ...) and the pid